starboard/android/shared/audio_renderer_passthrough.cc

```diff
@@ -592,6 +592,11 @@ void AudioRendererPassthrough::OnDecoderOutput() {
   auto decoded_audio = decoder_->Read(&decoded_audio_sample_rate);
   SB_DCHECK(decoded_audio);
 
+  if (decoded_audio->size_in_bytes() == 0) {
```

Comment:
Does this happen to be an end of stream output?  In that case we should use decoded_audio->is_end_of_stream();